packages/turbo/ios/RNVisitableView.swift

```diff
@@ -34,72 +65,64 @@ class RNVisitableView: UIView, SessionSubscriber {
   required init?(coder aDecoder: NSCoder) {
     super.init(coder: aDecoder)
   }
-  
-  public func getRNSesssion() -> RNSession? {
-    if (!Thread.isMainThread) {
-      print("getSession accessed from incorrect thread")
-      return nil
-    }
     
-    guard let sessionModule = self.bridge?.uiManager.moduleRegistry.module(forName: "RNVisitableViewModule") as? RNVisitableViewModule else {
-        print("Couldn't find session for sessionHandle:", sessionHandle ?? "default session")
-        return nil
-    }
-    return sessionModule.getSession(sessionHandle: sessionHandle)
-  }
-  
-  public func getSession() -> Session? {
-    return getRNSesssion()?.turboSession
+  public func getWebViewConfiguration() -> WKWebViewConfiguration{
+    let configuration = WKWebViewConfiguration()
+    configuration.applicationNameForUserAgent = applicationNameForUserAgent as String?
+    return configuration
   }
   
   override func didMoveToWindow() {
-
-    let url = URL(string: String(url))!
-    
-    if (self.controller == nil) {
-      print("Open new VistableView with URL passed from JS: ", url)
-      setupViewController(url: url)
-      self.addSubview((controller?.view)!)
+    if (!subviews.contains(controller.view)){
```

Comment:
Nitpicking, but is this conditional really needed? I suppose a single view can only be in the hierarchy once anyway.

Reply:
If we would go back, we would replace an old instance with a new one, but this shouldn't matter to much.